stable/grafana/templates/clusterrole.yaml

```diff
@@ -11,7 +15,11 @@ metadata:
   annotations:
 {{ toYaml . | indent 4 }}
 {{- end }}
+{{- if .Values.sidecar.useCurrentNamespace  }}
+  name: {{ template "grafana.fullname" . }}-role
+{{- else}}
   name: {{ template "grafana.fullname" . }}-clusterrole
+{{- end}}
 {{- if or .Values.sidecar.dashboards.enabled .Values.sidecar.datasources.enabled }}
```

Comment:
we can get rid of this conditional since it will always be true given the top-level one on Line 1

Reply:
done